src/components/publicApi/OtpRepairScreen.tsx

```diff
@@ -3,27 +3,30 @@ import * as React from 'react'
 
 import { useClearOnUnmount } from '../../hooks/useClearOnUnmount'
 import { Branding } from '../../types/Branding'
+import { OnLogEvent } from '../../types/ReduxTypes'
 import { Router } from '../navigation/Router'
 import { ReduxStore } from '../services/ReduxStore'
 
 interface Props {
   account: EdgeAccount
   branding: Branding
   context: EdgeContext
-  onComplete: () => void
   otpError: OtpError
+  onComplete: () => void
+  onLogEvent: OnLogEvent
```

Comment:
These need to be optional otherwise we have a breaking change to the login-ui API